debian/control

```diff
@@ -578,6 +578,29 @@ Description: Enhanced system logging daemon (xml parser plugin)
  The xml parser can process input in xml format, and adds the parsed data
  to the message object.
 
+Package: syslog-ng-mod-http
+Architecture: any
+Multi-Arch: foreign
+Depends: ${shlibs:Depends}, ${misc:Depends}, syslog-ng-core (>= ${source:Version}), syslog-ng-core (<< ${source:Version}.1~)
```

Comment:
I'm not 100% sure, but because this is just depends and not pre-depends, it might conflict on that file. I think the Breaks line is necessary in this case.

Reply:
I was just copy pasting this, so i accept your lead here. The point is that the http module should be in its own package.